lib/registrant.rb

```diff
@@ -0,0 +1,21 @@
+class Registrant
+    attr_reader :name, :age, :permit, :license_data
```

Comment:
We don't need/want `:permit` in the attr_reader since we have our method `permit?` which acts as our reader.